### PR TITLE
BookA-Pset_GeoObsFoldAxis

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_GeoObsFoldAxis

## Description: 

Properties for geological fold axis observations.

| Property | Datatype | Description |
|----------|----------|-------------|
|FoldType|IfcLabel|Types of Geological Folds, which can be classified according to various factors independently.|
|DeformationPhase|IfcLabel|Name or label for a geologic event such as a basin formation, compressionla texctonic phase, orogeny etc.|
|AzimuthHingeLine|IfcPlaneAngleMeasure|The azimuth of the hinge line of the fold. The azimuth is given in degrees in the range 0 to 360 degrees counted clockwise from true north and expressed as a three-digit number, e.g. 240 or 015.|
|AzimuthHingeLineation|IfcPlaneAngleMeasure|The azimuth of the lineation. The azimuth is given in degrees in the range 0 to 360 degrees counted clockwise from true north and expressed as a three-digit number, e.g. 240 or 015.|
|Plunge|IfcPlaneAngleMeasure|The angle of inclination of the fold axis with the horizontal as measured in a vertical plane.|

### CODE REVIEW

**Comment by patmclarin** [comment]
Please update description to: Properties for geological fold axis observations

File: IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The angle of inclination of the fold axis with the horizontal as measured in a vertical plane.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/Documentation.md]: Where is the vertical plane? There are an infinite number of them in a Cartesian CS.

File: IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Plunge_0jANEsqIjD28q6omcrafcL" Name="Plunge" UniqueId="2d2973b6-d12b-4d08-8d06-cb09b5929995" PropertyType="p_boundedvalue" PrimaryDataType="IfcPlaneAngleMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/DocProperty.xml]: Bounded value?

File: IFC4x3/Properties/a/AzimuthHingeLine_39UFUKWgf4gOAXmd1DKyEZ/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AzimuthHingeLine_39UFUKWgf4gOAXmd1DKyEZ" Name="AzimuthHingeLine" UniqueId="c978f794-82aa-44a9-82a1-c2704d53c3a3" PropertyType="p_boundedvalue" PrimaryDataType="IfcPlaneAngleMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AzimuthHingeLine_39UFUKWgf4gOAXmd1DKyEZ/DocProperty.xml]: Bounded value?

File: IFC4x3/Properties/a/AzimuthHingeLineation_1bK1xqeB57exCDB4QMcG7Z/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AzimuthHingeLineation_1bK1xqeB57exCDB4QMcG7Z" Name="AzimuthHingeLineation" UniqueId="65501ef4-a0b1-47a3-b30d-2c46969901e3" PropertyType="p_boundedvalue" PrimaryDataType="IfcPlaneAngleMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AzimuthHingeLineation_1bK1xqeB57exCDB4QMcG7Z/DocProperty.xml]: Bounded value?

File: IFC4x3/Properties/d/DeformationPhase_34v8HVl7D5$hjlWFWQDfkK/DocProperty.xml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DeformationPhase_34v8HVl7D5$hjlWFWQDfkK" Name="DeformationPhase" UniqueId="c4e4845f-bc73-45fe-bb6f-80f81a369b94" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/d/DeformationPhase_34v8HVl7D5$hjlWFWQDfkK/DocProperty.xml]: This one shall be re-used from Pset_GeoObsDiscontinuityFault, right?

File: IFC4x3/Properties/d/DeformationPhase_34v8HVl7D5$hjlWFWQDfkK/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Name or label for a geologic event such as a basin formation, compressionla texctonic phase, orogeny etc.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/d/DeformationPhase_34v8HVl7D5$hjlWFWQDfkK/Documentation.md]:
```suggestion
Name or label for a geologic event such as a basin formation, compressional tectonic phase, orogeny etc.
```

**Review by SergejMuhic — COMMENTED** [review]
Same as in other psets. The angle measure are incredibly confusing.

Type, see comment in https://github.com/bSI-InfraRoom/IFC-Specification/pull/600#pullrequestreview-1761310641

File: IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/DocProperty.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/DocProperty.xml]: > Bounded value?

Yes! It is specified that way.

File: IFC4x3/Properties/a/AzimuthHingeLine_39UFUKWgf4gOAXmd1DKyEZ/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/a/AzimuthHingeLine_39UFUKWgf4gOAXmd1DKyEZ/DocProperty.xml]: Yes! It is specified that way.

File: IFC4x3/Properties/a/AzimuthHingeLineation_1bK1xqeB57exCDB4QMcG7Z/DocProperty.xml (hunk 4 above)
Comment (larswik) [reply IFC4x3/Properties/a/AzimuthHingeLineation_1bK1xqeB57exCDB4QMcG7Z/DocProperty.xml]: > Bounded value?

Yes! It is specified that way.

File: IFC4x3/Properties/d/DeformationPhase_34v8HVl7D5$hjlWFWQDfkK/DocProperty.xml (hunk 5 above)
Comment (larswik) [reply IFC4x3/Properties/d/DeformationPhase_34v8HVl7D5$hjlWFWQDfkK/DocProperty.xml]: Can be done when Pset_GeoObsDiscontinuityFault is merged.

File: IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/Documentation.md (hunk 1 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/Documentation.md]: Proposal @j-weil and @larswik 
```suggestion
The angle of inclination of the fold axis with the horizontal as measured in a vertical (direction of gravity) plane.
```

Comment (larswik) [reply IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/Documentation.md]: > Proposal @j-weil and @larswik

LFTM!

**Comment by larswik** [comment]
> Same as in other psets. The angle measure are incredibly confusing.
> 
> Type, see comment in [#600 (review)](https://github.com/bSI-InfraRoom/IFC-Specification/pull/600#pullrequestreview-1761310641)

Observatioons has no type object and Reference is deprecated. Keep?

File: IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/Documentation.md (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/p/Plunge_0jANEsqIjD28q6omcrafcL/Documentation.md]: Updated according to proposal from domain.

**Comment by SergejMuhic** [comment]
> > Same as in other psets. The angle measure are incredibly confusing.
> > Type, see comment in [#600 (review)](https://github.com/bSI-InfraRoom/IFC-Specification/pull/600#pullrequestreview-1761310641)
> 
> Observatioons has no type object and Reference is deprecated. Keep?

Do we need to do anything about this still?

**Comment by larswik** [comment]
> > > Same as in other psets. The angle measure are incredibly confusing.
> > > Type, see comment in [#600 (review)](https://github.com/bSI-InfraRoom/IFC-Specification/pull/600#pullrequestreview-1761310641)
> > 
> > 
> > Observatioons has no type object and Reference is deprecated. Keep?
> 
> Do we need to do anything about this still?

I would prefer leaving the FoldType property.

**Pull request merged**